generators/app/templates/webapp/test/unit/unitTests.qunit.html

```diff
@@ -19,7 +19,7 @@
 		<link rel="stylesheet" type="text/css" href="../../resources/sap/ui/thirdparty/qunit-2.css" />
 		<script src="../../resources/sap/ui/thirdparty/qunit-2.js"></script>
 		<script src="../../resources/sap/ui/qunit/qunit-junit.js"></script>
-		<script src="../../resources/sap/ui/qunit/qunit-coverage.js"></script>
+		<script src="../../resources/sap/ui/qunit/<%= qunit-coverage-file %>"></script>
```

Comment:
`Error: qunit is not defined`
I think the variables must be valid JavaScript variables (not containing dashes). Existing variables use camelCase.

Reply:
Thx. Done.